expression/builtin_string_vec_test.go

```diff
@@ -504,6 +504,16 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 			newRealStringGener(),
 			&numStrGener{*newRangeInt64Gener(-10, 40)},
 		}},
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETDecimal, types.ETInt, types.ETString}, geners: []dataGenerator{
+			newRangeDecimalGener(-10000, 10000, 0.5),
+			newRangeInt64Gener(-10, 40),
+			newNullWrappedGener(0.5, &constStrGener{"en_US"}),
```

Comment:
`zh_CN` is also supported, we better write a generator function to generate locals, and it will be convenient to support new local formats in the future.

Reply:
https://github.com/pingcap/parser/blob/4cf2c2d4f51bdf72ef8ec493c6f94590c0870746/mysql/locale_format.go#L92-L94

it just returns "not implemented" for `zh_CN` and "not support for the specific locale" for other locale.

OK i'll try to write a locale generator. How to sync this generator when new locale is supported, left a code comment? Since `zh_CN` was contained in `locale2FormatFunction` but not implemented

https://github.com/pingcap/parser/blob/4cf2c2d4f51bdf72ef8ec493c6f94590c0870746/mysql/const.go#L739-L742

Reply:
I didn't noticed any difference between `zh_CN` and `en_US` on number representation [1]. Could I change [locale2FormatFunction](https://github.com/pingcap/parser/blob/4cf2c2d4f51bdf72ef8ec493c6f94590c0870746/mysql/const.go#L739-L742) to use same function for these two and use keys from `locale2FormatFunction` to write a locale generator? @b41sh @qw4990 

[1] https://en.wikipedia.org/wiki/Decimal_separator#Examples_of_use

Reply:
Sorry, I didn't see the details of `formatZHCN`, the local generator can only return `en_US` currently, we can add other formats in the future.
By the way, you can implement `formatZHCN` if you are interested